template/template_test.go

```diff
@@ -346,6 +346,31 @@ func TestRenderVersion(t *testing.T) {
 	}
 }
 
+func TestRenderUserDeactivation(t *testing.T) {
+	files := []string{"user.deactivation"}
+	reg := template.AssetRegistry{}
+	template, exist := reg.Get(string(types.UserDeactivation))
+	assert.True(t, exist)
+
+	for _, file := range files {
```

Comment:
why do you iterate on a single value?

Reply:
fixed [2ba0a97](https://github.com/fabric8-services/fabric8-notification/pull/104/commits/2ba0a97c27dbff57dfcc74cfca04e5d46ac24722)